### ISSUE TITLE
[Bug]: Comments do not download for videos requiring use of cookies

### ISSUE THREAD

**Issue by PhuriousGeorge** [tubearchivist/tubearchivist#596]
### I've read the documentation

- [X] I'm running the latest version of Tube Archivist and have read the [release notes](https://github.com/tubearchivist/tubearchivist/releases/latest).
- [X] I have read the [how to open an issue](https://github.com/tubearchivist/tubearchivist/blob/master/CONTRIBUTING.md#how-to-open-an-issue) guide, particularly the [bug report](https://github.com/tubearchivist/tubearchivist/blob/master/CONTRIBUTING.md#bug-report) section.

### Operating System

Unraid

## Describe the bug
Ref discussion: https://discord.com/channels/920056098122248193/1168888231329267782

Videos requiring cookies to download do not display comments.

It is assumed that the cookie is not passed to the comment-pulling portion of the script

## Steps To Reproduce
Download private or members-only video with comments enabled

## Expected behavior
Pull and display comments IAW user settings

### Relevant log output

```shell
[2023-11-13 20:24:36,574: WARNING/ForkPoolWorker-16] [task][check_reindex] reindex {'video': ['Yg1fdaGgtDE']}
[2023-11-13 20:24:36,698: WARNING/ForkPoolWorker-16] validating cookie
[2023-11-13 20:24:36,758: WARNING/ForkPoolWorker-16] Yg1fdaGgtDE: get metadata from es
[2023-11-13 20:24:36,761: WARNING/ForkPoolWorker-16] Yg1fdaGgtDE: get metadata from youtube
[2023-11-13 20:24:41,924: WARNING/ForkPoolWorker-16] UC76GYXW3j9z5g3OdW--Sf5g: get metadata from es
[2023-11-13 20:24:42,065: WARNING/ForkPoolWorker-16] Yg1fdaGgtDE: get ryd stats
[2023-11-13 20:24:42,537: WARNING/ForkPoolWorker-16] Yg1fdaGgtDE: get sponsorblock timestamps
[2023-11-13 20:24:43,068: WARNING/ForkPoolWorker-16] Yg1fdaGgtDE: sponsorblock failed: 404
[2023-11-13 20:24:43,069: WARNING/ForkPoolWorker-16] Yg1fdaGgtDE: delete subtitles
[2023-11-13 20:24:43,257: WARNING/ForkPoolWorker-16] Yg1fdaGgtDE-en: get user uploaded subtitles
[2023-11-13 20:24:43,257: WARNING/ForkPoolWorker-16] Yg1fdaGgtDE-en: get auto generated subtitles
[2023-11-13 20:24:48,134: WARNING/ForkPoolWorker-16] Yg1fdaGgtDE: get comments
[2023-11-13 20:24:49,683: WARNING/ForkPoolWorker-16] ERROR: [youtube] Yg1fdaGgtDE: Join this channel to get access to members-only content like this video, and other exclusive perks.
[2023-11-13 20:24:52,686: WARNING/ForkPoolWorker-16] 92603600-d633-4d10-8a93-98d7fb5253f6 success callback
[2023-11-13 20:24:52,687: INFO/ForkPoolWorker-16] Task check_reindex[92603600-d633-4d10-8a93-98d7fb5253f6] succeeded in 16.113426753989188s: 'reindexed 1 videos'
```

**Comment by bbilly1** [comment]
I think I know where this is going wrong, it's missing a config...

**Comment by bbilly1** [comment]
can somebody confirm that is working? Latest unstable build should have that fix included.

**Comment by bbilly1** [comment]
fixed in v0.4.4, thanks for helping with testing.
